runtime/gc_vlhgc/HeapRegionDataForAllocate.cpp

```diff
@@ -229,7 +221,40 @@ MM_HeapRegionDataForAllocate::addToArrayletLeafList(MM_HeapRegionDescriptorVLHGC
 	_previousArrayletLeafRegion = newSpineRegion;
 }
 
+#if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
 void 
+MM_HeapRegionDataForAllocate::pushRegionToLeafRegionList(MM_EnvironmentVLHGC *env, MM_HeapRegionDescriptorVLHGC **head)
```

Comment:
I'd rename this to push/popRegionFromArrayReservedRegionList

it's ok to reuse existing _nextArrayletLeafRegion  pointer (rather then introducing new one), even though the name does not fully fit